drivers/SmartThings/matter-switch/src/test/test_electrical_sensor.lua

```diff
@@ -0,0 +1,611 @@
+-- Copyright 2024 SmartThings
+--
+-- Licensed under the Apache License, Version 2.0 (the "License");
+-- you may not use this file except in compliance with the License.
+-- You may obtain a copy of the License at
+--
+--     http://www.apache.org/licenses/LICENSE-2.0
+--
+-- Unless required by applicable law or agreed to in writing, software
+-- distributed under the License is distributed on an "AS IS" BASIS,
+-- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+-- See the License for the specific language governing permissions and
+-- limitations under the License.
+
+local test = require "integration_test"
+local capabilities = require "st.capabilities"
+local t_utils = require "integration_test.utils"
+
+local clusters = require "st.matter.clusters"
+
+clusters.ElectricalEnergyMeasurement = require "ElectricalEnergyMeasurement"
+clusters.ElectricalPowerMeasurement = require "ElectricalPowerMeasurement"
+
+local mock_device = test.mock_device.build_test_matter_device({
```

Comment:
It might be good to add a test cases for a simple on/off plug that has power/energy measurement support

Reply:
good point